src/abstractdataframe/reshape.jl

```diff
@@ -150,16 +150,20 @@ melt(df::AbstractDataFrame; variable_name::Symbol=:variable, value_name::Symbol=
 Unstacks a DataFrame; convert from a long to wide format
 
 ```julia
-unstack(df::AbstractDataFrame, rowkey, colkey, value)
-unstack(df::AbstractDataFrame, colkey, value)
+unstack(df::AbstractDataFrame, rowkeys::Union{Symbol, Integer},
```

Comment:
`Union{Symbol, Integer}` -> `ColumnIndex`. Currently, `ColumnIndex` is actually `Union{Symbol, Real}`, but I've had a PR on my to-do list for a while to change it to `Union{Symbol, Integer}`. Let's stay consistent and hopefully we can further restrict the types of all column-accepting functions at once by just changing `ColumnIndex` to `Union{Symbol, Integer}`

Reply:
The problem is that it's not exported currently, and even if it was I'm not sure it's a good idea to use it in docs since it's less explicit than `Union`. Ideally we could interpolate it using `$ColumnIndex`.

Reply:
I have `Union{Symbol, Integer}` everywhere in my local repo added. I will probably push this during the weekend when I finish tuning the performance.

Reply:
If we use 4-space indent rather than backticks the variable interpolation works, just tried it out